lib/tokenize.cpp

```diff
@@ -8778,6 +8778,22 @@ void Tokenizer::simplifyKeyword()
                     }
                 }
             }
+
+            // noexcept -> noexcept(true)
+            // 3) void f() noexcept; -> void f() noexcept(true);
+            if (Token::Match(tok, ") noexcept [:{;]") ||
```

Comment:
You should be able to write a single `Token::Match`. Try this:

    Token::Match(tok, ") noexcept :|{|;|const|override|final")